installer/manifests/keptn/templates/core.yaml

```
@@ -169,8 +169,8 @@ spec:
               value: {{ ((.Values.features).automaticProvisioning).message | default "" | quote }}
             - name: AUTH_MSG
               value: {{ .Values.bridge.authMsg | default "" | quote }}
-            - name: D3_HEATMAP_ENABLED
-              value: {{ .Values.bridge.d3heatmap.enabled | default false | quote }}
+            - name: D3_ENABLED
+              value: {{ .Values.bridge.d3.enabled | default false | quote }}
```

I would suggest trying to be backward compatible if possible. So if a user had it enabled in the past it still works :) 
```suggestion
              value: {{ or .Values.bridge.d3heatmap.enabled ((.Values.bridge).d3).enabled | default false | quote }}
```